lib/game.rb

```diff
@@ -0,0 +1,64 @@
+require_relative './modules/gameutils'
+require_relative './player'
+require_relative './board'
+
+class Game
+  include GameUtils
+  attr_reader :player_one, :player_two, :active_player, :max_moves, :moves_made
```

Comment:
You are not using some of these accessors (:player_one, :player_two, :active_player,). If you don't plan to use them, make sure to remove them.

Reply:
We're using these accessors in the project.

Reply:
I checked that, it seems you are not using them. But I will check it again.